templates/calm/directory.tmpl

```diff
@@ -138,4 +142,8 @@ e-node=<img src="[websvn:locwebsvnhttp]/templates/calm/images/e-node.png" alt="[
   [websvn:compare_endform]
 </div>
 [websvn-endtest]
+[websvn-test:loadalldir]
+<script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/jquery.min.1.9.1.js"></script>
+<script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/collapsible.js"></script>
+[websvn-endtest]
```

Comment:
@ams-tschoening This is where the check for loadalldir is there. If it is set in config file then and only then the JS will be included. Otherwise it will be the same,